server/src/test/java/com/defold/extender/IntegrationTest.java

```diff
@@ -110,15 +110,18 @@ public static List<TestConfiguration> data() {
                 // "a" is a made up sdk where we can more easily test build.yml fixes
                 // new DefoldVersion("a", new Version(0, 0, 0), new String[] {"armv7-android", "x86_64-win32"} ),
```

Comment:
Enable it later when task with `platform.sdks.mapping.json` caching will be resolved.